pacehrh/vignettes/pacehrh.rmd

```diff
@@ -233,6 +242,18 @@ to the model. Columns G through I do not affect the running of the model, but ar
   with an individual. This value should typically range between 5 and 30 minutes. 
 * **HoursPerWeek**: The number of hours per week a healthcare workers spends on a task, usually a non-clinical task that is not directly related to population rates. For example, there may be a few hours of weekly administrative tasks required, regardless of clinical care provided. Community engagement, outreach, and supply management may also be tasks with this dynamic.
 
+### CoverageRates
+
+The **CoverageRates** sheets specify the service coverage rates for healthcare tasks from the beginning year of the simulation to the ending year of the simulation. The user has the flexibility to specify coverage rates for as many or few tasks as they prefer. Each row represents a healthcare task. Columns A and B reference the healthcare tasks as listed in **TaskValues** sheets. In columns C through W, the user enters any value between zero and one, which is used as a fixed coverage level for that task in the year indicated by the column name. The default sheet max out at 20 years, since extrapolation beyond that is a questionable choice. If a task is missing from the sheet, or the values are not filled in, the model will assume 100% coverage for that year-task combination. 
```

Comment:
The R code doesn't do anything to enforce this 20-year limit. Is that a problem? (The comment about the danger of simulating beyond 20 years might be enough.)

Reply:
Not enough of a problem to stop the merge.